symfony-reproducer.md

```diff
@@ -0,0 +1,111 @@
+   Be sure to add a test if possible as this will likely be asked of you during the review.
+4. Not sure how to fix? Are you able to create a failing test? Same as the above step, create a PR that demonstrates
+   the bug in a failing test. Even if it doesn't include a fix, it's highly likely it will get reviewed and confirmed.
+   You'll also likely get tips on how to fix!
+5. Open a [bug report issue](https://github.com/symfony/symfony/issues/new/choose) and clearly explain the issue
+   you are facing. Initially including a reproducer (as we'll build below) is a great way to make your issue stand out!
+
+> [!TIP]
+> In general, pull requests (like described in steps 3 and 4 above) are more likely to catch the eye
+> of maintainers than issues (even a PR that only includes a failing test).
+
+After you've created an issue (step 5 above), a maintainer (or even another contributor) will hopefully start
+a conversation to discuss the bug further. It's possible the bug is hard to describe or hard for maintainers to
+understand. A common thing they may ask is to _create a reproducer_. What's this? How do you do this? In my
+experience as a maintainer, when asking for a reproducer, this usually ends the discussion as it perhaps
+seems too hard or time-consuming for the contributor. This is not the case! Let's look at how to do this!
+
+## What is a Reproducer
+
+A reproducer is a standard, fresh, Symfony application that, aside from the initial Symfony app skeleton code,
+includes only the code to reproduce the bug you are seeing. Maintainers love these as it makes it super easy
+to see the same bug as you. They can clone it and test it locally to confirm and test possible fixes.
+
+> [!TIP]
+> Just the process of creating a reproducer is a great learning experience and helps narrow down the actual
+> bug. Many times, working on a reproducer will resolve your issue as you may find it's not a bug at all but
+> just a misconfiguration or misuse of a feature.
+
+## Create a Reproducer
+
+The easiest way to create a reproducer is using the [Symfony CLI](https://symfony.com/download):
+
+```bash
+symfony new --webapp my-reproducer
+```
+
+This new app will be using the latest stable version of Symfony. If you need to target
+a specific Symfony version, use the `--version` option:
+
+```bash
+symfony new --webapp --version=7.0 my-reproducer
+```
+
+If you are living on the edge and discovered a bug in the latest development version
+of Symfony, great! You can use `--version=next` to create the app.
+
+```bash
+symfony new --webapp --version=next my-reproducer
+```
+
+Now, open the new app in your IDE. You're now ready to add the code to recreate the the problem.
+This could be entities, services, messages, controllers, tests, etc. Remember, it's important
+to keep the code as simple as possible to make it easy for maintainers to see the exact code
+that causes the problem. Add all this as a single commit so you can reference both the repository
```

Comment:
I'd put it on a branch. Then it's ok if things get messy on multiple commits

Reply:
Yeah, fair enough. Should I perhaps remove this part? Feels like it could be a bit confusing to explain (they also might put their readme on the branch which we wouldn't want).

The main thing I wanted to enable, is an easy way for a maintainer to see the diff between a standard Symfony app and their code. I think, since `symfony new`, creates the initial commit, we're good.